src/tests/ftest/control/daos_server_mem_usage.py

```diff
@@ -0,0 +1,80 @@
+"""
+  (C) Copyright 2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import re
+from time import sleep
+
+from apricot import TestWithServers
+from run_utils import run_remote
+
+
+class DaosServerMemUsage(TestWithServers):
+    """Test memory usage after starting/stopping servers."""
+
+    def get_memory_data(self, description):
+        """Get the total, free, and available memory per server host.
+
+        Returns:
+            dict: total, free, and available memory per server host
+        """
+        memory_data = {}
+        result = run_remote(self.log, self.server_managers[0].hosts, "free -m", stderr=True)
+        if not result.passed:
+            self.fail(f'Error obtaining available memory on {result.failed_hosts}')
+        for data in result.output:
+            hosts = str(data.hosts)
+            try:
+                info = re.findall('Mem:\s+(\d+)\s+\d+\s+(\d+)\s+\d+\s+\d+\s+(\d+)', data.stdout)
```

Comment:
flake wants you to do this
```suggestion
                info = re.findall(r'Mem:\s+(\d+)\s+\d+\s+(\d+)\s+\d+\s+\d+\s+(\d+)', data.stdout)
```

Reply:
Fixed.